app/src/main/java/com/mapbox/dash/example/ui/SampleDriverNotificationView.kt

```diff
@@ -153,6 +185,61 @@ fun SampleDriverNotificationView(
             )
         }
 
+        is Incident -> {
+            DriverNotificationView(
+                modifier,
+                notification.textResource(),
+                notification.duration?.let { duration ->
+                    context.getString(
+                        R.string.dash_driver_notification_incident_description,
+                        duration.toString(DurationUnit.MINUTES, 0),
+                    )
+                },
+                notification.icon(),
+                0,
+                null,
+                R.string.dash_driver_notification_dismiss,
+                { uiState.onDismissClick(notification) },
+                notification.timeToDismiss,
+            )
+        }
+
+        is BetterEvRoute -> {
+            val resources = when (notification.betterRouteType) {
+                BetterEvRouteType.EXCLUDE_PLANNED_CHARGING -> Triple(
+                    SampleIcons.skipCharging,
+                    context.getString(R.string.dash_driver_notification_better_ev_route_skip_charging),
+                    context.getString(R.string.dash_driver_notification_better_ev_route_skip_charging),
+                )
+                BetterEvRouteType.INCLUDE_ADDITIONAL_CHARGING -> Triple(
+                    SampleIcons.chargingNeeded,
+                    context.getString(R.string.dash_driver_notification_better_ev_route_need_charging),
+                    context.getString(
+                        R.string.dash_driver_notification_better_ev_route_soc_details,
+                        notification.minDestinationSoc.toInt(),
+                    ),
+                )
+                else -> Triple(
+                    SampleIcons.fastAlternative,
+                    context.getString(R.string.dash_driver_notification_better_ev_route_general),
+                    context.getString(
+                        R.string.dash_driver_notification_better_ev_route_soc_details,
+                        notification.minDestinationSoc.toInt(),
+                    ),
+                )
+            }
+            DriverNotificationView(
+                modifier,
+                resources.second,
+                resources.third,
+                resources.first,
+                R.string.dash_driver_notification_show,
+                { uiState.onAcceptClick(notification) },
+                R.string.dash_driver_notification_dismiss,
+                { uiState.onDismissClick(notification) },
+            )
+        }
+
```

Comment:
Looks like this wasn't here before. If we forgot to migrate these examples from the internal repository, then let's keep them